### PR TITLE
[38593] Table highlighting in notification settings is wrong

### PR DESCRIPTION
**Pull request by bsatarnejad**
Class 'hover' is added to a column that has opHighLightCol attribute on it.

https://community.openproject.org/projects/openproject/work_packages/38593/activity

### CODE REVIEW

File: frontend/src/app/features/user-preferences/notifications-settings/table/notification-settings-table.component.html [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
       </colgroup>
-      <thead>
+      <thead class="-sticky">
```
Comment (HDinger) [inline frontend/src/app/features/user-preferences/notifications-settings/table/notification-settings-table.component.html]: Why did you add the `-sticky` class here? If I see that correct, it can't have any effect as the table itself isn't scrollable.

**Review by HDinger — COMMENTED** [review]
The bug has been fixed 👍 
I am just wondering about the `-sticky` class (see my comment below). If it is not needed, I would remove it.

File: frontend/src/app/features/user-preferences/notifications-settings/table/notification-settings-table.component.html (hunk 1 above)
Comment (bsatarnejad) [reply frontend/src/app/features/user-preferences/notifications-settings/table/notification-settings-table.component.html]: Yeah, for this bug it doesn't do anything. I added this class here to be the same as other tables like projects and workpackages tables.

Comment (HDinger) [reply frontend/src/app/features/user-preferences/notifications-settings/table/notification-settings-table.component.html]: For the workPackages and projects table it makes sense because these tables have an inner scrolling meaning that you can scroll the rows and the rest of the page remains static. In that case it makes sense to make the header sticky. Without that, it has no effect in the best case or unwanted site effects in the worst case. So please remove it from here. 
Afterwards it is good to merge 👍

Comment (bsatarnejad) [reply frontend/src/app/features/user-preferences/notifications-settings/table/notification-settings-table.component.html]: Ah yeah I got it, thanks for explanation.

**Review by HDinger — APPROVED** [review]
Thank you 👍 
Good to merge once the tests are green
